client/src/utils/mutations.js

```diff
@@ -19,6 +19,54 @@ export const LOGIN_USER = gql`
   }
 `;
 
+//adding bio
+export const ADD_BIO = gql`
+  mutation AddBio($userId: ID!, $bio: BioInput!) {
+    addBio(userId: $userId, bio: $bio) {
+      _id
+      username
+      age
+      aboutme
+      hobbies
+      gender
+      profilePicture
+      activityLevel
+    }
+  }
+`;
+
+//adding comment
+export const ADD_COMMENT = gql`
+  mutation AddComment($content: String!, $postId: ID!) {
+    addComment(content: $content, postId: $postId) {
+      _id
```

Comment:
You dont need ID because you will get it from the context argument on the backend